microservices/authorization/migrations/permissions/list/models/payment-stripe.json

```
@@ -63,15 +63,23 @@
           "admin": "allow"
         }
       },
-      "product": "payment-stripe.Product",
       "metadata": {
         "in": {
           "admin": "allow"
         },
         "out": {
           "admin": "allow"
         }
-      }
+      },
+      "customUnitAmount": {
+        "in": {
+          "user": "allow"
+        },
+        "out": {
+          "user": "allow"
```

```suggestion
          "admin": "allow"
        },
        "out": {
          "admin": "allow"
```

base microservices should not including something for users